pom.xml

```diff
@@ -629,17 +592,16 @@
     </reporting>
     <profiles>
         <profile>
-            <id>pmd</id>
+            <id>quickbuild</id>
             <activation>
                 <activeByDefault>false</activeByDefault>
             </activation>
-            <build>
-                <plugins>
-                    <plugin>
-                        <artifactId>maven-pmd-plugin</artifactId>
-                    </plugin>
-                </plugins>
-            </build>
+            <properties>
```

Comment:
❓ I see that these `properties` are defined outside the `activation` block whereas elsewhere you've used the `property` element (with `name` and `value` elements) inside the `activation` block. Are they equivalent?